.github/workflows/ingest-cadet-metadata.yml

```
@@ -74,30 +74,20 @@ jobs:
         run: time poetry run datahub ingest -c ingestion/create_cadet_databases.yaml
 
       - name: push metadata to datahub
+        id: push_datahub
         env:
           DATAHUB_GMS_TOKEN: ${{ secrets.DATAHUB_GMS_TOKEN }}
           DATAHUB_GMS_URL: ${{ vars.DATAHUB_GMS_URL }}
           DATAHUB_TELEMETRY_ENABLED: false
         run: time poetry run datahub ingest -c ingestion/cadet.yaml
 
       - name: Notify on failure
-        uses: slackapi/slack-github-action@v1.26.0
-        if: job.status == 'failure'
+        uses: slackapi/slack-github-action@v1.27.0
+        if: failure() && steps.push_datahub.outcome == 'failure'
```

```suggestion
        if: failure()
```

Maybe we want to alert on any failure actaully, not just the cadet ingestion?